genesis.md

```diff
@@ -481,3 +481,40 @@ same principle can be used to scale up other aspects of
 deployments, such as CPU, memory, or disk sizes — just keep in
 mind this may need to be done indirectly by specifying instance or
 disk types and adding the desired resources in your cloud-config.
+
+## Stemcell Version Upgrades
+
+Genesis provides a simple way for you to update stemcells for deployments based off of Genesis Kits.  The typical process is to upload a new stemcell to the BOSH Director and perform a `genesis deploy` on each deployment.
+
+In each kit there is a default for the os flavor and version:
+
+```
+params:
+  stemcell_os:      ubuntu-trusty
+  stemcell_version: latest
```

Comment:
This is turning into `3468.latest` for reasons of compiled stemcells.

Reply:
Updated to `3468.latest`